src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -17,22 +17,24 @@ namespace Microsoft.Docs.Build
 {
     internal class OpsConfigAdapter : IDisposable
     {
-        public static string ValidationServiceEndpoint =>
-            s_isProduction
-            ? "https://docs.microsoft.com"
-            : "https://ppe.docs.microsoft.com";
+        public static string ValidationServiceEndpoint => GetValue(
```

Comment:
with `switch expression`, we don't need this `GetValue` helper method and the code is more readable:

```suggestion
        public static string ValidationServiceEndpoint => env switch
        {
               prod => '',
               _ =>
        };
```

Reply:
We need overwrite method to the env, look at method: GetXrefMapApiEndpoint

Reply:
Maybe I can also change that.